terraform/vars/backend-staging.tfvars

```diff
@@ -0,0 +1,5 @@
+bucket         = "tf-state-zeno-rest-api"
+key            = "terraform/state/staging/terraform.tfstate"
```

Comment:
The `env:` prefix is something that terraform adds to non-default workspace state keys (https://developer.hashicorp.com/terraform/language/backend/s3#state-storage). So if we just specify `terraform.tfstate` here, the actual key in s3 will be `env:/{workspace_name}/terraform.tfstate` which may be cleaner (and less duplicative). We can also override `env:` with something else using the `workspace_key_prefix` attribute if we choose to.

Any changes here won't affect the production state file location.